package.json

```diff
@@ -41,6 +43,7 @@
         "nocache": "^2.0.0"
     },
     "devDependencies": {
+        "@chartiq/tslint-config-chartiq-base": "^0.1.0",
```

Comment:
@erik-ciq the other PR also had the airbnb base included as a dependency. Why would it be needed in the other one and not in this one? Should it be in both or neither?